src/aspire/config.ini

```diff
@@ -2,6 +2,7 @@
 # Whether to log any uncaught errors through a sys excepthook
 log_errors = 1
 cupy = 0
+normal_fft = 0 
```

Comment:
Don't make this an int/boolean if you're supporting more than 2 options. Keep it descriptive. Strings can be used in config.ini just as well. Don't use names like 'normal', but go with explicit names (e.g. `fft=pyfftw/cupy/scipy`)

Reply:
Done in commits 9714cf8 and a618bcb.